src/jdk.jlink/share/man/jmod.md

```diff
@@ -201,8 +202,7 @@ e.g. "2022-02-12T12:30:00-05:00".
       --cmds commands --config configfiles --header-files src/h
       --libs lib --main-class com.greetings.Main
       --man-pages man --module-version 1.0
-      --os-arch "x86_x64" --os-name "macOS"
-      --os-version "10.10.5" greetingsmod
+      --target-platform "macOS-aarch64" greetingsmod
```

Comment:
Can you change it to "macos-aarch64" as it is case sensitive.

Reply:
Thank you for spotting this. I updated both areas related to the target platform.